cmd/restic/cmd_copy.go

```diff
@@ -174,9 +174,12 @@ func similarSnapshots(sna *restic.Snapshot, snb *restic.Snapshot) bool {
 	return true
 }
 
+const numCopyWorkers = 8
+
 func copyTree(ctx context.Context, srcRepo restic.Repository, dstRepo restic.Repository,
-	visitedTrees restic.IDSet, rootTreeID restic.ID) error {
+	visitedTrees restic.IDSet, rootTreeID restic.ID, gopts GlobalOptions) error {
```

Comment:
The `gopts` parameter is still a left-over from before.

Reply:
Right. this was a leftover from a different patch i did which added a progress bar to restic copy (but the code i wrote added signifficant duplication so i decided to stash it)